tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist =  py27, py33, py36, py37
+envlist =  py26, py27, py33, py36
```

Comment:
I think this one doesn't need to be reverted... mind excluding this?

Reply:
Sorry for the delay there @alfredodeza - I've just repushed.